webvh/webvh/routes.py

```diff
@@ -0,0 +1,168 @@
+"""DID Webvh routes module."""
+
+from aiohttp import web
+from aiohttp_apispec import docs, querystring_schema, request_schema, response_schema
+from marshmallow import fields
+
+from acapy_agent.admin.decorators.auth import tenant_authentication
+from acapy_agent.admin.request_context import AdminRequestContext
+from acapy_agent.core.event_bus import Event, EventBus
+from acapy_agent.core.profile import Profile
+from acapy_agent.core.util import STARTUP_EVENT_PATTERN
+from acapy_agent.messaging.models.openapi import OpenAPISchema
+from acapy_agent.resolver.routes import ResolutionResultSchema
+from .did.exceptions import (
+    ConfigurationError,
+    DidCreationError,
+    DidUpdateError,
+    EndorsementError,
+)
+from .did.manager import DidWebvhManager
+
+
+class WebvhOptionsSchema(OpenAPISchema):
```

Comment:
is this request payload following a model we see with other did registration methods or is it specific to webvh?

Reply:
Umm. This is super basic right now specific to `webvh`. I was mostly waiting until I knew exactly what options would be available. Currently you just provide the namespace and an optional identifier.

It tries to be the same as what cheqd did with an options and features object, as this had been discussed briefly. It's very much open for changes.

Reply:
ok good to know. I would envision having a `parameters` key in there where the user can set some boolean flags for `prerotation` and `portable` as well as the witness parameters. The options key seems fine, maybe just change `features` with `parameters` ?

We should also discuss how we plan to handle versions of webvh which may or may not become a nightmare.